Updated docs for reconciler / controllers in autoscaling (#1982)
knative/docs: docs/serving/configuring-autoscaling.md

Knative uses a single shared autoscaler. This is, by default, the Knative Pod Autoscaler (KPA), which
provides fast, request-based autoscaling capabilities out of the box.

> You can also configure Knative to use Horizontal Pod Autoscaler (HPA), or use your own autoscaler, by creating a [controller](https://kubernetes.io/docs/concepts/architecture/controller/) (or [reconciler](https://godoc.org/k8s.io/kubernetes/pkg/controller/volume/attachdetach/reconciler)) for the Pod Autoscaler custom resource.

Reviewer: how about

```suggestion
You can also configure Knative to use Horizontal Pod Autoscaler (HPA), or use your own autoscaler, by creating a [controller](https://kubernetes.io/docs/concepts/architecture/controller/) (also referred to as a [reconciler](https://godoc.org/k8s.io/kubernetes/pkg/controller/volume/attachdetach/reconciler)) for the Pod Autoscaler custom resource.
```

You can implement your own Pod Autoscaler if the requirements of your workload cannot be covered by the KPA or HPA, for example if you want to use a more specialized autoscaling algorithm, or if you need to use a specialized set of metrics not supported by Knative out of the box.

> To implement your own Pod Autoscaler, you can create a controller that operates on your own class of Pod Autoscaler.

Reviewer: you "have to" create it right, its not an option?

```suggestion
To implement your own Pod Autoscaler, you must create a controller to handle your class of Pod Autoscaler.
```
